src/patternfly/components/DataList/data-list.scss

```diff
@@ -176,6 +182,30 @@
       font-size: var(--pf-c-data-list--m-compact__check--FontSize);
     }
   }
+
+  &.pf-m-truncate {
+    .pf-c-data-list__cell,
+    .pf-c-data-list__text {
```

Comment:
in a block like this, I don't think you need the `__cell` and `__text` selectors do you?

I also think you could write the sass like

```
.pf-c-data-list,
.pf-c-data-list__item-row,
.pf-c-data-list__cell,
.pf-c-data-list__text {
  &.pf-m-truncate {
    // truncate vars
  }
  .pf-m-break-word {
    // break-word vars
  }
  ...
}
```